readme/manual-setup.md

```diff
@@ -89,9 +90,48 @@ If you are doing development on Ozone and are building the Ozone distro in your
 ```bash
 export DISTRO_PATH=/your/path/to/ozone-distro/target/ozone-distro-$VERSION
 ```
+### Step 6. Setting up Traefik
```

Comment:
So I made this its own step in the setup, however, unlike with the quick start, it is never required to run this:
```bash
docker network create web
```
Is that intended?